### ISSUE TITLE
Polygon edge not found

### ISSUE THREAD

**Issue by elbamos** [ricardo-bion/ggradar#3]
Using the example from the readme:

```
> download.file("https://dl.dropboxusercontent.com/u/2364714/airbnb_ttf_fonts/Circular Air-Light 3.46.45 PM.ttf", "/Library/Fonts/Circular Air-Light 3.46.45 PM.ttf", method="curl")
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0100  216k  100  216k    0     0   491k      0 --:--:-- --:--:-- --:--:--  491k
> 
> extrafont::font_import(pattern = 'Circular', prompt=FALSE)
Scanning ttf files in /Library/Fonts/, /System/Library/Fonts, ~/Library/Fonts/ ...
Extracting .afm files from .ttf files...
/Library/Fonts/Circular Air-Light 3.46.45 PM.ttf => /Users/amos.elberg/Library/R/3.3/library/extrafontdb/metrics/Circular Air-Light 3.46.45 PM
Found FontName for 1 fonts.
Scanning afm files in /Users/amos.elberg/Library/R/3.3/library/extrafontdb/metrics
Writing font table in /Users/amos.elberg/Library/R/3.3/library/extrafontdb/fontmap/fonttable.csv
Writing Fontmap to /Users/amos.elberg/Library/R/3.3/library/extrafontdb/fontmap/Fontmap...
> library(ggradar)
> 
> suppressPackageStartupMessages(library(dplyr))
> library(scales)
> 
> mtcars %>%
+      add_rownames( var = "group" ) %>%
+      mutate_each(funs(rescale), -group) %>%
+      tail(4) %>% select(1:10) -> mtcars_radar
Warning message:
Deprecated, use tibble::rownames_to_column() instead. 
> 
> ggradar(mtcars_radar) 
Error in grid.Call(L_textBounds, as.graphicsAnnot(x$label), x$x, x$y,  : 
  polygon edge not found
```

Any suggestions?

**Comment by joseramoncajide** [comment]
Same here!
`> ggradar(mtcars_radar)
Error in grid.Call(L_textBounds, as.graphicsAnnot(x$label), x$x, x$y,  : 
  polygon edge not found`

**Comment by joseramoncajide** [comment]
Solved!. You must update ggplot2 to the latest version ;)

**Comment by teuschb** [comment]
Thank you. Strangely, running `update.packages()` worked, even though `ggplot2` wasn't one of the packages that needed updating.

**Comment by elbamos** [comment]
I had to re-install ggplot from github, but it works now.  Thanks!

**Comment by saurfang** [comment]
We solved this issue by restarting the computer.

**Comment by gstaniak** [comment]
For what it's worth, I saw this issue again yesterday. The environment is Mac OS X Catalina. Appeared suddenly, then was reproducible 100% (the only way to see _anything_ plotted was to use theme_void()), and there were the additional warnings about fonts. In retrospect, this happened in the middle of a session after I uninstalled a bunch of unneeded software from the Mac. I googled for the error, tried workarounds, upgraded R and RStudio to latest stable, nothing helped. I got rid of it by rebooting the computer. Looks like it may be system related, but who knows how.
